server/resources/user_experience.py

```diff
@@ -17,8 +14,23 @@ def post(self):
         exp = data["experience"].replace("\'", "\"")
         exp = json.loads(exp)
 
+        print(exp)
+
+        for key, value in exp.items():
+            print("key: " + key)
+            print("value: " + value)
+
+        # get user_id
+        user = UserModel.get_user(get_jwt_identity())
+
         try:
-            UserModel.update_experience(get_jwt_identity(), exp)
+            for key, value in exp.items():
+                new_exp = ExperienceModel(
+                    user_id=user.id,
+                    language=key,
+                    level=value
+                )
+                new_exp.save_to_db()
```

Comment:
we have to be a bit careful that we handle duplicates in this model (can't allow experience table with the same user id and language)

Reply:
Can you make the Experience Model's primary key a composite key of the user_id and language?

https://stackoverflow.com/questions/19129289/how-to-define-composite-primary-key-in-sqlalchemy/30067532
https://docs.sqlalchemy.org/en/13/core/constraints.html

Reply:
I was having trouble defining composite primary key to the user_id and language. 

For now though i added: __table_args__ = (db.UniqueConstraint("user_id", "language"),)   to the ExperienceModel, I believe this produces the same result where we cant have duplicates.